Parent/Parent/ParentAppDelegate.swift

```diff
@@ -94,31 +92,20 @@ class ParentAppDelegate: UIResponder, UIApplicationDelegate {
             let crashlyticsUserId = "\(session.userID)@\(session.baseURL.host ?? session.baseURL.absoluteString)"
             Firebase.Crashlytics.crashlytics().setUserID(crashlyticsUserId)
         }
-        legacySession = Session.current
         Analytics.shared.logSession(session)
         getPreferences()
         showRootView()
     }
 
     func showRootView() {
-        guard let session = legacySession else { return }
-        do {
-            let refresher = try Student.observedStudentsRefresher(session)
```

Comment:
If I'm understanding right, this refresher causes the #1 crash in Parent right now.